fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/HeaderNestingJsonBuilderStrategy.java

```diff
@@ -30,15 +38,27 @@ public HeaderNestingJsonBuilderStrategy(Function<Response, JsonNode> baseStrateg
 
     @Override
     public JsonNode apply(Response response) {
+        MappingJsonFactory mappingJsonFactory = new MappingJsonFactory();
         ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
         objectNode.set("response", baseStrategy.apply(response));
         try {
             objectNode.set(
                     "X-Druid-Response-Context",
-                    new MappingJsonFactory()
+                    mappingJsonFactory
                             .createParser(response.getHeader("X-Druid-Response-Context"))
                             .readValueAsTree()
             );
+            objectNode.set(
+                    "status-code",
+                    mappingJsonFactory.createParser(String.valueOf(response.getStatusCode())).readValueAsTree()
+            );
```

Comment:
@michael-mclawhorn Sure. This will happen once https://github.com/yahoo/fili/pull/275 is merged.